core/src/main/java/org/mule/api/MuleException.java

```diff
@@ -131,17 +131,22 @@ public final String getMessage()
     protected void initialise()
     {
         setExceptionCode(ExceptionHelper.getErrorCode(getClass()));
-        String javadoc = ExceptionHelper.getJavaDocUrl(getClass());
-        String doc = ExceptionHelper.getDocUrl(getClass());
-        if (javadoc != null)
+        if(DefaultMuleConfiguration.docLinks)
```

Comment:
There is no need to make this conditional, we don't want to include this info anymore